recipes/ctng-compilers/build.sh

```diff
@@ -0,0 +1,126 @@
+#!/bin/bash
+
+set -x
+
+mkdir -p .build/src
+mkdir -p .build/tarballs
+
+if [[ $(uname) == Darwin ]]; then
+  DOWNLOADER="curl -SL"
+  DOWNLOADER_INSECURE=${DOWNLOADER}" --insecure"
+  DOWNLOADER_OUT="-C - -o"
+else
+  DOWNLOADER="wget -c"
+  DOWNLOADER_INSECURE=${DOWNLOADER}" --no-check-certificate"
+  DOWNLOADER_OUT="-O"
+fi
+
+mkdir -p ${SYS_PREFIX}/conda-bld/src_cache/
+# Some kernels are not on kernel.org, such as the CentOS 5.11 one used (and heavily patched) by RedHat.
+if [[ ! -e "${SYS_PREFIX}/conda-bld/src_cache/linux-${ctng_kernel}.tar.bz2" ]] && \
+   [[ ! -e "${SYS_PREFIX}/conda-bld/src_cache/linux-${ctng_kernel}.tar.xz" ]]; then
+  if [[ ${ctng_kernel} == 2.6.* ]]; then
+    ${DOWNLOADER} ftp://ftp.be.debian.org/pub/linux/kernel/v2.6/linux-${ctng_kernel}.tar.bz2 ${DOWNLOADER_OUT} ${SYS_PREFIX}/conda-bld/src_cache/linux-${ctng_kernel}.tar.bz2
+  elif [[ ${ctng_kernel} == 3.* ]]; then
+    # Necessary because crosstool-ng looks in the wrong location for this one.
+    ${DOWNLOADER} https://www.kernel.org/pub/linux/kernel/v3.x/linux-${ctng_kernel}.tar.bz2 ${DOWNLOADER_OUT} ${SYS_PREFIX}/conda-bld/src_cache/linux-${ctng_kernel}.tar.bz2
+  elif [[ ${ctng_kernel} == 4.* ]]; then
+    ${DOWNLOADER} https://www.kernel.org/pub/linux/kernel/v4.x/linux-${ctng_kernel}.tar.xz ${DOWNLOADER_OUT} ${SYS_PREFIX}/conda-bld/src_cache/linux-${ctng_kernel}.tar.xz
+  fi
+fi
+
+# Necessary because uclibc let their certificate expire, this is a bit hacky.
+if [[ ${ctng_libc} == uClibc ]]; then
+  if [[ ! -e "${SYS_PREFIX}/conda-bld/src_cache/uClibc-${ctng_uClibc}.tar.xz" ]]; then
+    ${DOWNLOADER_INSECURE} https://www.uclibc.org/downloads/uClibc-${ctng_uClibc}.tar.xz ${DOWNLOADER_OUT} ${SYS_PREFIX}/conda-bld/src_cache/uClibc-${ctng_uClibc}.tar.xz
+  fi
+else
+  if [[ ! -e "${SYS_PREFIX}/conda-bld/src_cache/glibc-${gnu}.tar.bz2" ]]; then
+    ${DOWNLOADER_INSECURE} https://ftp.gnu.org/gnu/libc/glibc-${gnu}.tar.bz2 ${DOWNLOADER_OUT} ${SYS_PREFIX}/conda-bld/src_cache/glibc-${gnu}.tar.bz2
+  fi
+fi
+
+# Necessary because CentOS5.11 is having some certificate issues.
```

Comment:
Is this needed? Or just inherited from an old recipe when CentOS 5 was supported?

Reply:
I'd like to keep this recipe close to AnacondaRecipes which uses this.